### ISSUE TITLE
Empty list of splits on config-split-names

### ISSUE THREAD

**Issue by AndreaFrancis** [huggingface/dataset-viewer#2803]
We have some entries with empty splits:
```
Atlas atlas-x5jgb3-shard-0 [primary] datasets_server_cache> db.cachedResponsesBlue.countDocuments({kind:"config-split-names", http_status:200, "content.splits":{ $exists: true, $size: 0}})
97

```
Note that the viewer for these datasets always shows: `Error code:   ConfigNotFound `
Like https://huggingface.co/datasets/Vinay573/aarkoocustomdataset
![image](https://github.com/huggingface/dataset-viewer/assets/5564745/a6db1910-1ed7-433b-9c25-63a7854a350f)
But, when using the datasets library, it is possible to get the dataset split:
![image](https://github.com/huggingface/dataset-viewer/assets/5564745/3a8a4557-931c-42c0-9d16-459d1ce903db)
This issue is due because we first load splits from [config-info](https://github.com/huggingface/dataset-viewer/blob/main/services/worker/src/worker/job_runners/config/split_names.py#L154) and then from streaming, but if config-info is empty, it will store no splits.
Should we throw an error if the splits list is empty?

**Comment by severo** [comment]
Yes, sure, it does not make sense for the dataset viewer to process a dataset without splits. Better to raise an error

**Comment by severo** [comment]
ok, fixed
